src/main/java/seedu/duke/CommandList.java

```diff
@@ -1,10 +1,11 @@
 package seedu.duke;
 
+import seedu.duke.stats.MatchStat;
 import seedu.duke.ui.Ui;
 import seedu.duke.ai.Ai;
 
 public enum CommandList {
-    BYE, SHOOT, PENALTY
+    BYE, SHOOT, PENALTY, YES, NO
```

Comment:
You can give us more information when these two command can be called

Reply:
These two commands are called when one match ends.

Reply:
1. If user wants to leave in the middle of the match, he can just enter bye.
2. User cannot choose to play single round games.